index.d.ts

```diff
@@ -15,6 +15,15 @@ declare namespace cpFile {
 		@default 0o777
 		*/
 		readonly directoryMode?: number;
+
+		/**
+		Base directory to find the source file and copy it to this directory.
+
+		`cwd` works by being prepended with source, destination path.
```

Comment:
That's an implementation detail. It doesn't matter for the user. You should explain how it works from the user's perspective.